README.md

```diff
@@ -27,7 +27,7 @@ reaches on any path of the story should not have any options.  For an example, s
 [this file](https://github.com/udacity/create-your-own-adventure/blob/c4f2bf7caac641df1c979a2db056a532e0c93e23/english/light-fire/fire.md)
 from an early version of the story.  Here are some examples of  appropriate changes:
 
-* Add a sentence or two to an existing "page" (file) of the story.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.